apps/portal/src/content/docs/components/drawer.mdx

```diff
@@ -65,20 +200,250 @@ import { Drawer } from '@harnessio/ui/components'
 //...
 
 return (
-  <Drawer.Root>
-    <Drawer.Trigger>Open</Drawer.Trigger>
+  <Drawer.Root
+    direction="left"
+    open={isDrawerOpen}
+    onOpenChange={setIsDrawerOpen}
+  >
+    <Drawer.Trigger asChild>
+      <Button>Open drawer</Button>
+    </Drawer.Trigger>
+
+    <Drawer.Content size="md" hideClose>
+      <Drawer.Header icon="info">
+        <Drawer.Tagline>Section</Drawer.Tagline>
+        <Drawer.Title>Title of the drawer</Drawer.Title>
+        <Drawer.Description>Description of the drawer</Drawer.Description>
+      </Drawer.Header>
+
+      <Drawer.Body className="">
+        Content of the drawer goes here
+
+        <Drawer.Root nested>
+          <Drawer.Trigger asChild>
+            <Button>Open Nested Drawer</Button>
+          </Drawer.Trigger>
+
+          <Drawer.Content>
+            <Drawer.Header>
+              <Drawer.Title>Title</Drawer.Title>
+            </Drawer.Header>
+
+            <Drawer.Body>
+              Content
+            </Drawer.Body>
+
+            <Drawer.Footer>
+              Footer
+            </Drawer.Footer>
+          </Drawer.Content>
+        </Drawer.Root>
+      </Drawer.Body>
+
+      <Drawer.Footer>
+        <ButtonGroup>
+            <Drawer.Close asChild>
+              <Button variant="outline">Cancel</Button>
+            </Drawer.Close>
+            <Button>Submit</Button>
+        </ButtonGroup>
+      </Drawer.Footer>
+    </Drawer.Content>
+  </Drawer.Root>
+)
+```
+
+## Anatomy
+
+All parts of the `Alert` component can be imported and composed as required.
+
+```typescript jsx
+<Drawer.Root>
+    <Drawer.Trigger />
```

Comment:
We only need to use 4-space indent in the `ExampleComponent` blocks.
```suggestion
```typescript jsx
<Drawer.Root>
  <Drawer.Trigger />
```